pontoon/machinery/utils.py

```diff
@@ -56,6 +59,29 @@ def get_google_translate_data(text, locale_code):
         }
 
 
+def get_concordance_search_results(text, locale):
+    search_phrases = base.utils.get_search_phrases(text)
+    search_filters = (
+        Q(Q(target__icontains_collate=(phrase, locale.db_collation)), locale=locale,)
```

Comment:
We should add `| Q(source__icontains=phrase),` here. According to the [spec](https://github.com/mozilla/pontoon/blob/a4f6e09/specs/0106-concordance-search.md#feature-explanation), CS "searches source strings and translations".